src/screens/LocationDetailsScreen.tsx

```diff
@@ -1,19 +1,364 @@
+        </View>
+        <View style={styles.chartContainer}>
+          <VictoryChart width={420} height={175}>
+            <VictoryAxis
+              // tickValues specifies both the number of ticks and where
+              // they are placed on the axis
+              // TODO: Will need to change this to be dynamic based on the hours of operation
+              tickValues={[
+                7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
+              ]}
+              tickFormat={[
+                "7am",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "3pm",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "11pm",
+              ]}
+              style={{
+                tickLabels: { fill: theme.styles.placeholderText.color },
+                axis: { stroke: "none" },
+              }}
+            />
+            <VictoryBar
+              style={{
+                data: {
+                  fill: "#1677FF",
+                  opacity: ({ datum }) =>
+                    datum.quarter === poiData.currentHour ? 1 : 0.5,
+                },
+              }}
+              cornerRadius={{ top: 3, bottom: 3 }}
+              data={chartData}
+              x="quarter"
+              y="earnings"
+            />
+          </VictoryChart>
+        </View>
+      </ScrollView>
+      <View style={styles.waitTimeContainer}>
+        <View>
+          <StyledText style={styles.waitTimeText}>Wait Time</StyledText>
+          <StyledText style={styles.detailsText}>
+            {"Current time is " +
+              poiData.waitTime +
+              (poiData.waitTime === 1 ? " min" : " mins") +
+              " • " +
+              renderLastUpdated(poiData.lastUpdated)}
+          </StyledText>
+        </View>
+        <View style={styles.confirmWaitTimeButtonContainer}>
+          <Button
+            style={[theme.styles.confirmWaitTimeButton, styles.waitTimeButtons]}
+            onPress={confirmNewWaitTime}
+          >
+            <StyledText
+              style={[
+                theme.styles.primaryColor,
+                styles.confirnWaitTimeButtonText,
+              ]}
+            >
+              {"Confirm " +
+                poiData.waitTime +
+                (poiData.waitTime === 1 ? " min" : " mins")}
+            </StyledText>
+          </Button>
+        </View>
+        <View style={styles.newWaitTimeContainer}>
+          <View style={styles.newWaitTimeButtonsContainer}>
+            <Button
+              size="large"
+              style={[
+                theme.styles.newWaitTimePlusMinusButton,
+                styles.waitTimeButtons,
+                styles.newWaitTimePlusMinusButton,
+                styles.newWaitTimeMinusButton,
```

Comment:
This is confusing, why so many styles for this one button. And using "new" doesn't really help.

Reply:
`new` is not for like new its meant for `new` wait time. I tried to modularize it because a lot of the styles need to be reused with other buttons and plus/minus button

Reply:
okay could some of the name be changed then to describe what the style is instead of the component cause `newWaitTimePlusMinusButton` doesn't really help.